### PR TITLE
Controller board new config

### PR DESCRIPTION
**Pull request by ywang282**
as title

### CODE REVIEW

File: controller/controller-board/src/app/components/services/ClusterConfigService.js [hunk 1]
```diff
@@ -13,24 +13,25 @@
         var rawClusterConfig = {};
         var runningHosts = [];
         var dataSegments = [];
-        var hostsFromConfig = {};
         var shardConfig = {};
         var hostInConfig = [];
-        var hostNotInConfig = [];
+        var blacklistedHosts = [];
+        var candidateHosts = [];
```
Comment (shuzhang1989) [inline controller/controller-board/src/app/components/services/ClusterConfigService.js]: can we call them "healthyStandbyHosts?"

File: controller/controller-board/src/app/views/host.html [hunk 2]
```diff
@@ -107,10 +103,31 @@
             </md-tab>
             <md-tab md-on-select="onTabSelected(tab)" md-on-deselect="announceDeselected(tab)">
                 <md-tab-label>
-                    Dead Hosts
+                    Candidate Hosts
```
Comment (shuzhang1989) [inline controller/controller-board/src/app/views/host.html]: Healthy Standby Hosts

**Comment by ywang282** [comment]
This is the update on the configs, the replace host diff will be build on top of this diff

screen shot 2017-11-15 at 7 14 19 pm


screen shot 2017-11-15 at 7 16 48 pm

screen shot 2017-11-15 at 7 17 06 pm

**Comment by shuzhang1989** [comment]
great!

**Pull request merged**